### DISCUSSION TITLE
Outpost Security wont stop inspecting eachother after Jailbreak

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14569] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After triggering the alarm on a jailbreak mission, outpost security NPCs turn to Team2 and keep inspecting eachother if they're not fighting

![image](https://github.com/user-attachments/assets/ba4cb64c-81c2-494b-8ff4-57d6a2d79207)

### Reproduction steps

Do a jailbreak mission
Trigger alarm
Teleport to your sub
Watch the mandatory genital inspection unfold

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build!
